src/views/social/Facebook.js

```diff
@@ -0,0 +1,10 @@
+import '../../assets/css/socialLogin.scss';
+import { FACEBOOK_CALLBACK_URL } from '../../config/index';
+import facebookIcon from '../../assets/images/facebook_32px.png';
+import socialLoginButton from '../../components/socialLoginButton';
+
+const Facebook = () => (
+  socialLoginButton(FACEBOOK_CALLBACK_URL, facebookIcon, 'log in with Facebook', 'Signup with Facebook')
+);
+
+export default Facebook;
```

Comment:
The same here, make it reusable like the way I suggest on the Google component file